spec/tic_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require '../lib/tic'
+require '../lib/players'
```

Comment:
- [ ]  Using `require_relative` for files in your directory instead of `require` is a ruby convention. Always use `require_relative` to import local files.